bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/rulesupport/internal/LoaderScriptExtension.java

```diff
@@ -81,14 +84,17 @@
         staticTypes.put("TriggerType", TriggerType.class);
         staticTypes.put("ConfigDescriptionParameter", ConfigDescriptionParameter.class);
 
+        staticTypes.put("ItemRegistry", null);
+        staticTypes.put("ir", null);
```

Comment:
As I understood, the DefaultScriptScopeProvider (the whole load mechanism) is obsolete in the near future, so I did not add it there.